### ISSUE TITLE
Accessibility: High Contrast:  Iconbutton does not follow high contrast standards on Firefox

### ISSUE THREAD

**Issue by twarzo** [OfficeDev/office-ui-fabric-react#3916]
### Bug Report
- __Browser and OS versions__: Firefox Version 58.0.1 (64-bit), Windows 10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
No

Requested priority:
Normal

Products/sites affected: (if applicable)
VSTS

#### Describe the issue:
IconButton is not following high contrast color standards in Firefox browser

#### Actual behavior:
In HCB, the Icon button has Cyan background and white color instead of Black  background and white color. 
![image](https://user-images.githubusercontent.com/2266283/35958593-9b02ffb8-0cc7-11e8-8f65-6d1e62aa5435.png)

In HCW, the Icon button has Violet background and white color instead of White background and black color. 
![image](https://user-images.githubusercontent.com/2266283/35958693-15f91086-0cc8-11e8-85fc-8b31b423ad81.png)

#### Expected behavior:
As per High Contrast Black Theme:
Buttons should be Black in color with blue background.
As per High Contrast White Theme:
Buttons should be White in color with violet background.

**Comment by jordandrako** [comment]
We currently only support IE and Edge HIgh Contrast, as other browser implementations are inconsistent.
